Live/Classes/HUD/HUD.h

```diff
@@ -0,0 +1,33 @@
+#ifndef _HUD_HPP_
+#define _HUD_HPP_
+
+#include "cocos2d.h"
+#include "HUDBar.h"
+#include "Player.h"
+
+class HUD : public cocos2d::Layer {
+public:
+    // HUD stores a reference to game objects that have the potential to update the HUD
+    HUD(Player* player);
```

Comment:
does HUD have any control over the player? like can it change player's attributes? If not, store as a const.